Rakefile

```diff
@@ -23,7 +23,11 @@ webpacked_js = "javascript/main.js"
 installed_package_lock_json = "node_modules/.package-lock.json"
 
 file installed_package_lock_json => ["package.json", "package-lock.json"] do
-  sh("npm", "install", "--no-save")
+  if production?
+    sh("npm", "ci")
+  else
+    sh("npm", "install", "--no-save")
+  end
```

Comment:
What is the difference of `npm ci` and `npm install --no-save`?

Reply:
https://docs.npmjs.com/cli/v7/commands/npm-ci#description

> This command is similar to npm install, except it's meant to be used in automated environments such as test platforms, continuous integration, and deployment -- or any situation where you want to make sure you're doing a clean install of your dependencies.

Reply:
Thanks.